spec/models/institution_spec.rb

```diff
@@ -185,4 +185,40 @@ def create_institution(institution, description)
       end
     end
   end
+
+  describe 'when exporting' do
+    let(:mapping) { described_class::CSV_CONVERTER_INFO }
+
+    # Pull the default CSV options to be used
+    default_options = Rails.application.config.csv_defaults[described_class.name] ||
+                      Rails.application.config.csv_defaults['generic']
+    # Merge with provided options
+    load_options = default_options.each_with_object({}) { |(k, v), o| o[k.to_sym] = v; }
```

Comment:
I'm a bit confused by what you mean when you say

> Merge with provided options

It kind of looks like you're just converting the hash keys from strings to symbols, e.g.:
```
{"skip_lines"=>0, "col_sep"=>","}
  ...becomes...
{:skip_lines=>0, :col_sep=>","}
```

Reply:
In which case, you may want to use `default_options.symbolize_keys!`

Reply:
fixed
copy paste error from `spec/models/shared_examples/shared_examples_for_loadable.rb`

institutions table is not created via CSV import but during the process in `app/models/institution_builder.rb` however without a test for exporting we missed the issue that is being fixed

Reply:
Was able to remove that line as was not necessary, it's used in shared_examples_for_loadable to merge provided options with default options but here it's not needed